<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="05068-020L" name="testCases.TS_001_Sales_Planning" tests="25" failures="3" timestamp="28 Mar 2019 10:58:29 GMT" time="16.157" errors="1">
  <testcase name="TC_003_Verify_District_Lable" time="1.927" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_005_Verify_GnDivision_Lable" time="0.058" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_014_select_districtValue" time="1.307" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_009_Verify_Remarks_Lable" time="0.036" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_004_Verify_TerriTory_Lable" time="0.092" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_017_verify_select_StarttDate" time="0.221" classname="testCases.TS_001_Sales_Planning">
    <error type="org.openqa.selenium.WebDriverException" message="unknown error: Element &lt;input type=&quot;text&quot; name=&quot;sdate&quot; class=&quot;date-range-without-time form-control&quot; id=&quot;sdate&quot; value=&quot;2019-03-28&quot; data-date-startdate=&quot;2019-03-28&quot;&gt; is not clickable at point (1364, 343). Other element would receive the click: &lt;div style=&quot;position: absolute; top: 0; left: 0; width: 100%; height: 100%; text-align: center; z-index: 1000; background-color: #000; opacity: 0.2;&quot;&gt;&lt;/div&gt;
  (Session info: chrome=73.0.3683.86)
  (Driver info: chromedriver=2.42.591088 (7b2b2dca23cca0862f674758c9a3933e685c27d5),platform=Windows NT 10.0.14393 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: &#039;3.14.0&#039;, revision: &#039;aacccce0&#039;, time: &#039;2018-08-02T20:05:20.749Z&#039;
System info: host: &#039;05068-020L&#039;, ip: &#039;172.22.101.88&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_191&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.42.591088 (7b2b2dca23cca0..., userDataDir: C:\Users\SUPUN_~1\AppData\L...}, cssSelectorsEnabled: true, databaseEnabled: false, goog:chromeOptions: {debuggerAddress: localhost:52294}, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 73.0.3683.86, webStorageEnabled: true}
Session ID: be66cd2a5410045cfb23c4ed0bc89f73">
      <![CDATA[org.openqa.selenium.WebDriverException: unknown error: Element <input type="text" name="sdate" class="date-range-without-time form-control" id="sdate" value="2019-03-28" data-date-startdate="2019-03-28"> is not clickable at point (1364, 343). Other element would receive the click: <div style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; text-align: center; z-index: 1000; background-color: #000; opacity: 0.2;"></div>
  (Session info: chrome=73.0.3683.86)
  (Driver info: chromedriver=2.42.591088 (7b2b2dca23cca0862f674758c9a3933e685c27d5),platform=Windows NT 10.0.14393 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:05:20.749Z'
System info: host: '05068-020L', ip: '172.22.101.88', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_191'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.42.591088 (7b2b2dca23cca0..., userDataDir: C:\Users\SUPUN_~1\AppData\L...}, cssSelectorsEnabled: true, databaseEnabled: false, goog:chromeOptions: {debuggerAddress: localhost:52294}, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 73.0.3683.86, webStorageEnabled: true}
Session ID: be66cd2a5410045cfb23c4ed0bc89f73
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)
	at org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40)
	at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80)
	at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:276)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:83)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
	at com.sun.proxy.$Proxy8.click(Unknown Source)
	at proxy.verifications.Selectdate(verifications.java:158)
	at pages.SalesPlanningPage.verifyselectStartdate(SalesPlanningPage.java:224)
	at testCases.TS_001_Sales_Planning.TC_017_verify_select_StarttDate(TS_001_Sales_Planning.java:184)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TC_017_verify_select_StarttDate -->
  <testcase name="TC_010_VerifyIs_District_DropdownEnabled" time="0.047" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_020_verify_Confirm_message" time="0.709" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_013_VerifyIs_Towers_DropdownEnabled" time="0.050" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_0181_verify_select_EndDate" time="0.491" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_008_Verify_Tower_Lable" time="0.056" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_019_verify_Click_tower_Check_Box" time="1.085" classname="testCases.TS_001_Sales_Planning">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertFalse(Assert.java:63)
	at org.testng.Assert.assertFalse(Assert.java:73)
	at proxy.verifications.ClickonCheckBox(verifications.java:171)
	at pages.SalesPlanningPage.ClickonToweCheckBox(SalesPlanningPage.java:254)
	at testCases.TS_001_Sales_Planning.TC_019_verify_Click_tower_Check_Box(TS_001_Sales_Planning.java:202)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC_019_verify_Click_tower_Check_Box -->
  <testcase name="TC_002_Verify_navigateTo_SalesPlanning" time="5.823" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_022_verify_SubmitSuccessFully" time="0.002" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_012_VerifyIs_GNDivisions_DropdownEnabled" time="0.040" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="verifyDistrictvalues" time="0.598" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="close" time="0.001" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_006_Verify_StartDate_Lable" time="0.040" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_014_districtValue" time="0.065" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_016_Verify_select_GN_Division" time="1.544" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_007_Verify_EndDate_Lable" time="0.050" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_011_VerifyIs_Territory_DropdownEnabled" time="0.033" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_015_select_Terriotorry" time="0.909" classname="testCases.TS_001_Sales_Planning">
    <failure type="java.lang.AssertionError" message="expected [COL - Colombo Depot] but found [COL - Colombo Depot
 ]">
      <![CDATA[java.lang.AssertionError: expected [COL - Colombo Depot] but found [COL - Colombo Depot
 ]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at proxy.verifications.verifyGlobalSelectDropDown(verifications.java:123)
	at pages.SalesPlanningPage.verifySelectedTerriotarry(SalesPlanningPage.java:180)
	at testCases.TS_001_Sales_Planning.TC_015_select_Terriotorry(TS_001_Sales_Planning.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC_015_select_Terriotorry -->
  <testcase name="verifyTerritoryvalues" time="0.606" classname="testCases.TS_001_Sales_Planning"/>
  <testcase name="TC_021_verify_Cancle_Submission" time="0.367" classname="testCases.TS_001_Sales_Planning">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertFalse(Assert.java:63)
	at org.testng.Assert.assertFalse(Assert.java:73)
	at pages.SalesPlanningPage.ClickOnCancleButton(SalesPlanningPage.java:290)
	at testCases.TS_001_Sales_Planning.TC_021_verify_Cancle_Submission(TS_001_Sales_Planning.java:227)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC_021_verify_Cancle_Submission -->
</testsuite> <!-- testCases.TS_001_Sales_Planning -->
